kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -211,6 +213,9 @@ organisms:
   {{- if .truncateColumnDisplayTo }}
   truncateColumnDisplayTo: {{ .truncateColumnDisplayTo }}
   {{- end }}
+  {{- if .columnWidth }}
+  columnWidth: {{ .columnWidth }}
+  {{- end }}
```

Comment:
Could you please also document this in the Helm chart config?

Reply:
Done (I hadn't seen it in a search because it was `.mdx` not `.md`) - though I agree with Cornelius that HTML isn't a good format to maintain this in

Reply:
Thanks! (Yes, it would definitely be nice to create a proper schema with autogenerated documentation but until then, this is the simplest way to have something.)